### PR TITLE
Update documentation with new Submit script feature

### PR DESCRIPTION
**Pull request by karlaspuldaro**
Following up to #1330

Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by karlaspuldaro** [comment]
I forgot to push the doc update commit to my previous PR, oops 🤭

File: docs/source/getting_started/overview.md [hunk 1]
```diff
@@ -51,14 +51,16 @@ Learn more about the Pipeline Visual Editor in the [AI Pipelines topic](/user_gu
 
 The pipeline editor feature can optionally be [installed as a stand-alone extension](installation).
 
-#### Ability to run a notebook as a batch job
+#### Ability to run a notebook or Python script as a batch job
 
-Elyra extends the Jupyter Notebook UI to to allow for execution of a notebook as a batch job in local or remote cloud environments. This feature leverages the AI pipelines feature and requires access to a Kubeflow Pipelines deployment via a [runtime configuration](../user_guide/runtime-conf).
+Elyra extends the Jupyter Notebook UI to to allow for execution of a notebook as a batch job in local or remote cloud environments. This feature leverages the AI pipelines feature and requires either a Kubeflow Pipeline or Airflow deployment via a [runtime configuration](../user_guide/runtime-conf).
```
Comment (ptitzler) [inline docs/source/getting_started/overview.md:56]:
```suggestion
Elyra extends the Jupyter Notebook UI to to allow for execution of a notebook as a batch job in local or remote cloud environments. This feature leverages the AI pipelines feature and requires either a Kubeflow Pipelines or Apache Airflow deployment via a [runtime configuration](../user_guide/runtime-conf).
```

File: docs/source/user_guide/enhanced-python-support.md [hunk 2]
```diff
@@ -23,18 +23,30 @@ these scripts in remote environments.
 
 ![Enhanced Python Support](../images/python-editor.gif)
 
-The execution of the python script leverages the available `Python based Kernels`. This enables
+The execution of the Python script leverages the available `Python based Kernels`. This enables
 users to run their scripts in different configurations and environments.
 
+Elyra also allows submitting a Python script as a single node pipeline for execution in a Kubeflow or Airflow environment in the cloud. This feature is accessible when the Elyra [AI Pipelines](../user_guide/pipelines.md) extension is also enabled.
```
Comment (ptitzler) [inline docs/source/user_guide/enhanced-python-support.md:29]:
```suggestion
Elyra also allows submitting a Python script as a single node pipeline for execution in a Kubeflow Pipelines or Apache Airflow environment in the cloud. This feature is accessible when the Elyra [AI Pipelines](../user_guide/pipelines.md) extension is also enabled.
```

File: docs/source/user_guide/enhanced-python-support.md [hunk 3]
```diff
@@ -23,18 +23,30 @@ these scripts in remote environments.
...
 ## Python script execution support
 
 * In the Jupyter Lab Launcher, click the `Python File` icon to create a new Python Script.
 * When used in conjunction with `Jupyter Enterprise Gateway`, the dropdown will be populated with more kernel options,
 allowing users to run their scripts with remote kernels with more specialized resources.
 * To run your script locally, select the `python3` option in the dropdown menu, and click the `Run` icon.
 
+## Ability to run a Python script as a batch job
+
+Elyra allows the execution of a Python script as a batch job in remote cloud environments. This feature leverages the AI pipelines feature and requires access to either a Kubeflow Pipelines or Airflow deployment via a [runtime configuration](../user_guide/runtime-conf).
```
Comment (ptitzler) [inline docs/source/user_guide/enhanced-python-support.md:40]:
```suggestion
Elyra allows the execution of a Python script as a batch job in remote cloud environments. This feature leverages the AI pipelines feature and requires access to either a Kubeflow Pipelines or Apache Airflow deployment via a [runtime configuration](../user_guide/runtime-conf).
```

File: docs/source/user_guide/enhanced-python-support.md [hunk 4]
```diff
@@ -23,18 +23,30 @@ these scripts in remote environments.
...
+
+To submit a Python script as a batch job, open a Python script file and select `Submit Script...` button from the editor toolbar.
+
+![Submit Python Script as a batch job](../images/submit-script.gif)
+
+To learn more about pipeline [runtimes configuration]((../user_guide/runtime-conf)) and [runtime images]((../user_guide/runtime-image-conf)), follow those sections provided in the Elyra user guide documentation.
```
Comment (ptitzler) [inline docs/source/user_guide/enhanced-python-support.md:46]:
```suggestion
To learn more about [runtime configurations](../user_guide/runtime-conf) and [runtime images](../user_guide/runtime-image-conf), follow those sections provided in the Elyra user guide documentation.
```

**Comment by ptitzler** [comment]
According to GitHub the images are zero bytes in size ):

**Comment by karlaspuldaro** [comment]
Thanks for reviewing this @ptitzler ! I'm not sure why the zero bytes on the new images, if you can still click on `display rich diff` and preview them as expected

**Review by lresende — APPROVED** [review]

**Review by ptitzler — APPROVED** [review]
LGTM

**Pull request merged**